packages/lib/src/vis/shared/hooks.ts

```diff
@@ -84,3 +84,12 @@ export function useZoomOnWheel(
 
   return onWheel;
 }
+
+export function useCameraFOV() {
+  const camera = useThree((state) => state.camera);
+
+  const { x: right, y: top } = CAMERA_TOP_RIGHT.clone().unproject(camera);
+  const { x: left, y: bottom } = CAMERA_BOTTOM_LEFT.clone().unproject(camera);
+
+  return { left, right, top, bottom };
```

Comment:
If the hook is used in a component that doesn't also call `useFrameRendering`, then the FOV will be stale pretty quickly. Perhaps it would be better to return a function created with `useCallback` - i.e `useGetCameraFOV`, thus following the same pattern as `useMoveCameraTo`?

Alternatively, you could make it a normal function with a single `camera` parameter and call it from inside your `useCallback` in `SelectionMesh`.

Reply:
Might be worth using this new utility in `PanMesh`, by the way!

Reply:
> If the hook is used in a component that doesn't also call useFrameRendering, then the FOV will be stale pretty quickly. 

Good point ! I will change this